batavia/types/Bool.js

```diff
@@ -299,7 +307,7 @@ Bool.prototype.__pow__ = function(other) {
             }
```

Comment:
At column 46: [(eqeqeq) Expected '===' and instead saw '=='.](http://eslint.org/docs/rules/eqeqeq)